### ISSUE TITLE
support response json array

### ISSUE THREAD

**Issue by funcfoo** [liyasthomas/postwoman#805]
**Describe the bug**
![](https://i.imgur.com/dN126Sg.png)
**To Reproduce**
1. Go to 'response'
2. If response is JSON array
3. See error

**Expected behavior**
support JSON array

**Desktop (please complete the following information):**
 - OS: OSX10.15.4
 - Browser: Chrome81.0.4044.113

**Comment by liyasthomas** [comment]
Please use the issue template for god's sake.
Can you at least provide a test endpoint so that we could replicate issue at our end.

**Comment by funcfoo** [comment]
sorry about that.

To Reproduce
Steps to reproduce the behavior:
1. import this curl: 
```
curl -X GET "https://my-json-server.typicode.com/typicode/demo/posts" -H "accept: application/json"
```
2. Go to 'response'
3. See error

**Comment by shtakai** [comment]
Hello. :sushi: 

The type it seems `{}` as `Object` by `this.response.body.constructor.name`.
But in this case, it seems `[]` as `Array` by  `this.response.body.constructor.name`.

I tried changing `lang` on Editor component's prop.
- `json` => displays error `x`.
- `json5` => displays no errors.

So I modified by using `json5` when the response body is `Array`.
(`json` when is is `Object`)

![Screenshot from 2020-04-26 13-30-24](https://user-images.githubusercontent.com/2187465/80297878-3c1a8e80-87c2-11ea-8100-7f7854d8012b.png)
![Screenshot from 2020-04-26 13-30-51](https://user-images.githubusercontent.com/2187465/80297882-4472c980-87c2-11ea-9228-d081eebc732b.png)


I'll make PR later.

Thank you.
